과제3/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>디자인과제3</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+<link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="box">
+        <div class="header">
+            <img src="navigation.png" class="img-nav" alt="">
+            <p class="title">Machiatto</p>
+            <img src="cart.png" class="img-cart" alt="">
+        </div>
+            <button class="heart">
+                <img src="heart.png" width="24px" height="24px" alt="">
+                <p class="heart248">248</p>
+            </button>
+        <div class="main">
+            <img src="coffee-main.png" class="img-main" alt="">
+            <p class="Machiatto">Machiatto</p>
+            <p class="text-1">Ma Kira To (Machiatto), the espresso add two<br>
+                tablespoons of milk into a cup of macciato. Macdo<br>
+                is the brand mark meaning in Italian, so a symbol<br>
+                of sweet imprint.</p>
+            <p class="price">$ 5.88</p>
+        </div>
+        <div class="menu">
+            <label for="radio1" class="radio">
+                <img src="coffee-1.jpg" class="img-coffee" alt="">
+                <input type="radio" name="radio" id="radio1">
+            </label>
+            <label for="radio2" class="radio">
+                <img src="coffee-2.jpg" class="img-coffee" alt="">
+                <input type="radio" name="radio" id="radio2">
+            </label>
+            <label for="radio3" class="radio">
+                <img src="coffee-3.jpg" class="img-coffee" alt="">
+                <input type="radio" name="radio" id="radio3">
+            </label>
+            <label for="radio4" class="radio">
+                <img src="coffee-4.jpg" class="img-coffee"alt="">
+                <input type="radio" name="radio" id="radio4">
+            </label>
+        </div>
+        <div class="location">
+            <img src="location.png" class="img-location" alt="">
+            <p class="text-2">Room 201, No. 2, Holzer, Melbourne, A..</p>
```

Comment:
27번 줄 리뷰와 마찬가지로 text-2보다는 location-description 이 더 좋을 것 같습니다!
```suggestion
            <p class="location-description">Room 201, No. 2, Holzer, Melbourne, A..</location-description>
```